doc-content/enterprise-only/openshift/operator-deploy-security-proc.adoc

```diff
@@ -31,6 +31,12 @@ If you selected `LDAP`, configure LDAP authentication:
 
 .. In the *LDAP URL* field, enter the LDAP URL.
 .. Configure LDAP parameters that correspond to the settings of the LdapExtended Login module of {EAP}. For instructions about using these settings, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html/login_module_reference/login_modules_with_external_identity_store#ldapextended_login_module[LdapExtended Login Module].
++
+[NOTE]
+====
+If you want to enable LDAP failover, you can set two or more LDAP server addresses in the `AUTH_LDAP_URL` parameter, separated by the space character.
```

Comment:
Minor suggestion - Change article of space and remove "character". IBMSG appears to refer to the space character as "space" or "a space". I tried rewriting the note, please disregard if it doesn't convey the information correctly.

s/If you want to enable LDAP failover, you can set two or more LDAP server addresses in the `AUTH_LDAP_URL` parameter, separated by the space character./To enable LDAP failover, set two or more LDAP server addresses in the `AUTH_LDAP_URL` parameter with each address separated by a space.